### DISCUSSION TITLE
Message type icons

### DISCUSSION THREAD

**Discussion by Blatman** [tauri-apps/tauri#5365] [Q&A]
MacOS 12.6 Monterey - Tauri 1.1.1

Been trying the message dialogs and the icon types (info/warning/error) in Prod don't seem to line up with the icon type descriptions.
info/warning just give the standard app icon and the error seems to be more of a warning style. In Dev the folder shown is to be expected. Is it possible that the icons could be sourced from the standard src-tauri folder which would then allow easy customisation?

![messagetypes](https://user-images.githubusercontent.com/11089335/194647767-9ec6c1af-fb92-4c49-9b54-aba65fd34520.png)

**Comment by FabianLars** [comment]
as far as i know there's no way to set custom icons in these dialogs. We can only tell macOS which application it's supposed to show and it creates the icons itself.

**Reply by Blatman** [reply]
mmmm - ok it looks as though the app icon is being identified OK and in the case of the error type it has created a composite icon albeit with the warning icon rather than error and for the info/warning it just uses the app icon as the code can't locate the system info and warning icons to make a composite. I guess the issue lies further into the webview code.
